.luarc.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This is an unrelated change but fixes the "unknown `vim` global" errors that appear when developing in this repo 😸

Reply:
This is the recommended way to configure the language server in a workspace: https://luals.github.io/wiki/configuration/#luarcjson-file

But okay, I understand. I'll remove this commit.